app/services/login/after_login_actions.rb

```diff
@@ -56,11 +56,9 @@ def check_id_mismatch(identity_value, mpi_value, error_message)
     end
 
     def create_mhv_account
```

Comment:
Can you add a check to ensure user is loa3? Otherwise we're gonna get a ton of sidekiq job failures because 'ICN must be present'